app/models/image.rb

```diff
@@ -1,2 +1,3 @@
 class Image < ActiveRecord::Base
+  belongs_to :performer
```

Comment:
this association should probably be polymorphic since we'll eventually have images belonging to performers, users, and films BUT i don't think you hit that requirement yet, so as this is, i think it's good for now 👍